client-ui/src/components/pages/auth/commonAuth.css

```diff
@@ -0,0 +1,18 @@
+body{
```

Comment:
Use this formatting in all css rules
```suggestion
body {
```